packages/plugins/rum/src/validate.ts

```diff
@@ -141,3 +153,26 @@ export const validatePrivacyOptions = (options: Options): ToReturn<PrivacyOption
 
     return toReturn;
 };
+
+export const validateSourceCodeContextOptions = (
+    options: OptionsWithDefaults,
+): ToReturn<SourceCodeContextOptions> => {
+    const red = chalk.bold.red;
+    const validatedOptions: RumOptions = options[CONFIG_KEY] || {};
+    const toReturn: ToReturn<SourceCodeContextOptions> = {
+        errors: [],
+    };
+
+    if (!validatedOptions.sourceCodeContext) {
+        return toReturn;
+    }
+
+    const cfg = validatedOptions.sourceCodeContext as SourceCodeContextOptions;
+
+    if (!cfg?.service || typeof cfg.service !== 'string') {
+        toReturn.errors.push(`Missing ${red('"sourceCodeContext.service"')}.`);
```

Comment:
To avoid any confusion from user's standpoint.

```suggestion
        toReturn.errors.push(`Missing ${red('"rum.sourceCodeContext.service"')}.`);
```